api/cloudprotocol/monitoring.go

```diff
@@ -47,21 +47,21 @@ type MonitoringData struct {
 	InTraffic  uint64           `json:"inTraffic"`
 	OutTraffic uint64           `json:"outTraffic"`
 	Disk       []PartitionUsage `json:"disk"`
+	Timestamp  time.Time        `json:"timestamp"`
 }
 
 // NodeMonitoringData node monitoring data.
 type NodeMonitoringData struct {
 	MonitoringData
-	NodeID           string                   `json:"nodeId"`
-	Timestamp        time.Time                `json:"timestamp"`
-	ServiceInstances []InstanceMonitoringData `json:"serviceInstances"`
```

Comment:
Also leave `ServiceInstances` for core update only json tag. But `items` looks ambiguous.